tests/unit/type/list.tcl

```diff
@@ -315,6 +447,60 @@ start_server {
         assert_equal {foo} [r lrange blist 0 -1]
     }
 
+    test "BRPOPRPUSH with wrong destination type" {
```

Comment:
for these 3 i also don't see any value in repeating.
i think we can trim them (and replace the `after 1000` in the original one).

Reply:
Done.